packages/api/api-generator/src/commands/generate/generateCrud/generate-crud.ts

```diff
@@ -412,6 +412,14 @@ function generateService({ generatorOptions, metadata }: { generatorOptions: Cru
 }
 
 function generateEntityImport(targetMetadata: EntityMetadata<any>, relativeTo: string): Imports[0] {
+    const libMatch = targetMetadata.path.match(/(packages\/api|@comet)\/cms-api\/lib\/(.*)/);
```

Comment:
- `.../packages/api/cms-api/lib` for demo
- .`../@comet/cms-api/lib` for real projects